### PR TITLE
[1LP][RFR] Move convenience methods to use SystemdService plugin for appliance

### PR DESCRIPTION
**Pull request by mshriver**
{{ pytest: --long-running cfme/tests/ansible/test_embedded_ansible_crud.py  }}

There is some flaky behavior in the new tests checking for some EmbeddedAnsible process. @sbulage is going to address these test failures. I've changed the fixture name here, but that's unrelated to the process state changes that cause the test to fail.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit bfe5a02dffe8c58eadc3040a6323d13d4d96bfdd:

`cfme/tests/ansible/test_embedded_ansible_crud.py`:
- :bangbang: Line 74:70: **E127** *continuation line over-indented for visual indent*
- :heavy_exclamation_mark: Line 135:30: **W292** *no newline at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by sbulage** [comment]
LGTM at First sight.

File: cfme/tests/ansible/test_embedded_ansible_crud.py [hunk 1]
```diff
@@ -11,7 +12,7 @@
 
 
 @pytest.fixture(scope='module')
-def enabled_embedded_appliance(appliance):
+def embedded_appliance(appliance):
```
Comment (digitronik) [inline cfme/tests/ansible/test_embedded_ansible_crud.py]: Optional: 
but best place of this fixture is `global` as per me. what you think?... lets take advantage of this fixture to other contributes and may be same FA owner as well.

Comment (mshriver) [reply cfme/tests/ansible/test_embedded_ansible_crud.py]: Sure, not going to move it in this PR though :)

Certainly, defining basic fixtures to be consumed globally would be much more efficient.

**Comment by izapolsk** [comment]
good idea to turn those in services.  PR looks good but PRT failed due to sprout. I restarted PRT

**Pull request merged**